ntcore/src/main/java/edu/wpi/first/networktables/Publisher.java

```diff
@@ -2,12 +2,7 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
-#include "networktables/RpcCall.h"
+package edu.wpi.first.networktables;
 
-#include "networktables/NetworkTableEntry.h"
-
-using namespace nt;
-
-NetworkTableEntry RpcCall::GetEntry() const {
-  return NetworkTableEntry{m_entry};
-}
+/** NetworkTables publisher. */
+public interface Publisher extends PubSub {}
```

Comment:
Why does git think this file is a rename?

Reply:
No idea... nothing we can do about that.